examples/snippets/modules/ROOT/examples/base-images-override/.scalafmt.conf

```diff
@@ -0,0 +1,44 @@
+version = "2.3.2"
```

Comment:
is the `scalafmt` configuration all the same across the repo?
was, this one, missing?

Reply:
They are not the same across the repo, but I'm pretty sure they are the same for all the samples. This one was copied from one of the other samples.

Reply:
thanks!